target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq6018-fap650.dts

```diff
@@ -0,0 +1,458 @@
+// SPDX-License-Identifier: MIT, GPL-2.0 or later
+/* Copyright (c) 2023, Ruslan Isaev <legale.legale@gmail.com> */
+
+/dts-v1/;
+
+
+#include "ipq6018.dtsi"
+#include "ipq6018-cp-cpu.dtsi"
+#include "ipq6018-ess.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	model = "Yuncore FAP650";
+	compatible = "yuncore,fap650", "qcom,ipq6018-cp03", "qcom,ipq6018";
+	
+	/delete-node/ rpm-glink;
+	
+	aliases {
+		ethernet0 = &dp5;
+		ethernet1 = &dp4;
+		ethernet2 = &dp3;
+		ethernet3 = &dp2;
+		ethernet4 = &dp1;
+		label-mac-device = &dp5;
+		serial0 = &blsp1_uart3;
+		led-boot = &led_system;
+		led-failsafe = &led_system;
+		led-running = &led_system;
+		led-upgrade = &led_system;
+	};
+
+	chosen {
+		stdout-path = "serial0:115200n8";
+		bootargs-append = " root=/dev/ubiblock0_1 earlycon debug iomem=relaxed";
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&tlmm 19 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_system: led_system {
+			label = "system";
+			color = "red";
+			gpios = <&tlmm 32 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_wlan2g {
+			label = "wlan2g";
+			color = "green";
+			gpios = <&tlmm 37 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_wlan5g {
+			label = "wlan5g";
+			color = "blue";
+			gpios = <&tlmm 35 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+	reserved-memory {
+		#address-cells = <2>;
+		#size-cells = <2>;
+		ranges;
+
+		rpm_msg_ram: memory@60000 {
+			reg = <0x0 0x00060000 0x0 0x6000>;
+			no-map;
+		};
+
+		tz: memory@4a600000 {
+			reg = <0x0 0x4a600000 0x0 0x00400000>;
+			no-map;
+		};
+
+		smem_region: memory@4aa00000 {
+			reg = <0x0 0x4aa00000 0x0 0x00100000>;
+			no-map;
+		};
+
+		q6_region: memory@4ab00000 {
+			reg = <0x0 0x4ab00000 0x0 0x03700000>;
+			no-map;
+		};
+
+		nss_region: nss@40000000 {
+			no-map;
+			reg = <0x0 0x40000000 0x0 0x01000000>;
+		};
+
+		uboot@4A100000 {
+			no-map;
+			reg = <0x0 0x4a100000 0x0 0x00400000>;
+		};
+
+		sbl@4a500000 {
+			no-map;
+			reg = <0x0 0x4a500000 0x0 0x00100000>;
+		};
+
+		q6_etr_region: q6_etr_dump@1 {
+			no-map;
+			reg = <0x0 0x4e200000 0x0 0x00100000>;
+		};
+
+		m3_dump_region: m3_dump@50100000 {
+			no-map;
+			reg = <0x0 0x4e300000 0x0 0x00100000>;
+		};
+
+	};
+
+	memory {
+		device_type = "memory";
+		reg = <0x0 0x40000000 0x0 0x20000000>;
+	};
+
+};
+
+&soc {
+	q6v5_m3: q6v5_m3 {
```

Comment:
Is this just copy/paste from QSDK?

Because, I dont think that its parsed at all in the upstream kernel

Reply:
It is from qsdk. Remove?

Reply:
It is. Do you think need to be removed?

Reply:
Yeah, remove it